src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -73,6 +73,18 @@ PARAM_DEFINE_FLOAT(MPC_THR_MIN, 0.12f);
  */
 PARAM_DEFINE_FLOAT(MPC_THR_HOVER, 0.5f);
 
+/**
+ * Hover thrust source selector
+ *
+ * Set false to use the fixed parameter MPC_THR_HOVER
+ * (EXPERIMENTAL) Set true to use the value computed by the hover thrust estimator
+ *
+ * @boolean
+ * @max 0
```

Comment:
@dagar Does that work? Since this is experimental, I would like to have to click on "force save" to enable it.

Reply:
Let's change the category.